insights/parsers/lpstat.py

```diff
@@ -87,3 +90,32 @@ def printer_names_by_status(self, status):
         """
         names = [prntr['name'] for prntr in self.printers if prntr['status'] == status]
         return names
+
+
+@parser(Specs.lpstat_protocol_printers)
+class LpstatProtocol(CommandParser, dict):
+    """
+    Class to parse ``lpstat -v`` command output.
+
+    Sample output of the command::
```

Comment:
One blank line is required after the `::` section

Reply:
Updated